api/lib/application/authentication/oidc/oidc-controller.js

```diff
@@ -12,4 +15,28 @@ module.exports = {
 
     return h.response({ redirectLogoutUrl }).code(200);
   },
+
+  async findUserForOidcReconciliation(request) {
```

Comment:
```suggestion
  async findUserForReconciliation(request) {
```

On se trouve déjà dans l'oidc controller.

Reply:
si on décide ça, on l'applique partout du coup, mais ça me va